src/games/strategy/engine/framework/startup/ui/editors/EmailSenderEditor.java

```diff
@@ -39,8 +39,6 @@
   private final JCheckBox m_useTLS = new JCheckBox("Use TLS encryption");
   private final JTextField m_password = new JPasswordField();
   private final JLabel m_toLabel = new JLabel("To:");
-  private final JLabel m_loginLabel = new JLabel("Login:");
```

Comment:
Again no. Class level variables add state to the class, we should be minimizing that.